### ISSUE TITLE
Problem with web interface (port: 26000)

### ISSUE THREAD

**Issue by mehdighninoudev** [jtpereyda/boofuzz#606]
### Report

Hi I have launched boofuzz to test my vm but when I want to connect to web interface http://127.0.0.1:26000 I got 500 Internal server error
I am using boofuzz 0.41 from pip install boofuzz (python3.6)

Can you help me to resolve the issue please ?

### Steps to reproduce the problem

1.
2.
3.

**Comment by jtpereyda** [comment]
Hi @mehdighninoudev . You might try checking out the console for debug output -- I expect you should be able to find some more error output.

**Comment by Captain-Kludge** [comment]
I just downloaded the tool and think it's really cool! I am able to fuzz via command line, but see the same error in the browser.

(This occurred on both my Windows 10 machine and Kali Linux machine as well)

Here is the console errors I am seeing: 
![fuzz](https://user-images.githubusercontent.com/20678500/162189658-09c046b1-1167-4d06-b815-5782ae7c9ee6.jpg)

**Comment by jtpereyda** [comment]
Is this error specific to the `boo open` command for you?

**Comment by Captain-Kludge** [comment]
'boo fuzz' will also generate the first error about 'curses', but i'm probably not using that command correctly since it says i'm missing a command

**Comment by jtpereyda** [comment]
Yes that one is just a warning. Sometimes `boo open` can fail if you open a file created with a different version of boofuzz -- is that a possible issue here?

**Comment by Captain-Kludge** [comment]
no this db file was created by this same version just 5 minutes prior to the error :(

**Comment by jtpereyda** [comment]
Would you be able to share a script and/or steps to reproduce the problem?

**Comment by Captain-Kludge** [comment]
sure! though now I'm having problems recreating the problem, despite not touching this for a few days, so let me check back in a bit here :)

**Comment by Captain-Kludge** [comment]
Without changing anything, I'm now having trouble recreating the issue :)

will keep updated once I have more info!
